examples/example-jwt-auth/README.md

```diff
@@ -0,0 +1,65 @@
+Authentication Example
+==============================================
+
+This example illustrates a simple JWT-based authentication implementation in gRPC using
+ server interceptor. It uses the JJWT library to create and verify JSON Web Tokens (JWTs).
+
+The example requires grpc-java to be pre-built. Using a release tag will download the relevant binaries
+from a maven repository. But if you need the latest SNAPSHOT binaries you will need to follow
+[COMPILING](../../COMPILING.md) to build these.
+
+The source code is [here](src/main/java/io/grpc/examples/jwtauth). Please follow the
+[steps](../README.md#-to-build-the-examples) to build the examples. The build creates scripts
+`auth-server` and `auth-client` in the `build/install/examples/bin/` directory which can be
+used to run this example. The example requires the server to be running before starting the
+client.
+
+Running auth-server is similar to the normal hello world example and there are no arguments to supply:
+
+**auth-server**:
+
+```text
+USAGE: AuthServer
+```
+
+The auth-client accepts optional arguments for user-name and client-id:
+
+**auth-client**:
+
+```text
+USAGE: AuthClient [user-name [client-id]]
+```
+
+The `user-name` value is simply passed in the `HelloRequest` message as payload and the value of
+`client-id` is included in the JWT claims passed in the metadata header.
+
+
+#### How to run the example:
+
+```bash
+# Run the server:
+./build/install/example-jwt-auth/bin/auth-server
```

Comment:
This does not work for me either (after using my modified build steps). I get:
```
./build/install/example-jwt-auth/bin/auth-server
Error: Could not find or load main class io.grpc.examples.authentication.AuthServer
Caused by: java.lang.ClassNotFoundException: io.grpc.examples.authentication.AuthServer
```

I found multiple problems (at least):
- the `./build/install/example-jwt-auth/bin/auth-server` script uses this CLASSPATH definition
```
CLASSPATH=$APP_HOME/lib/example-jwt-auth.jar:...
```
I find the jar files in `libs` and NOT `lib`. And the other jar files referenced in CLASSPATH don't even exist under $APP_HOME (the current example dir).

Then looking at the jar file in the current libs directory:
```
jar tvf build/lib/example-jwt-auth.jar
...
 2487 Fri Sep 06 12:47:28 PDT 2019 io/grpc/examples/jwtauth/AuthServer.class
...
```
SO that's the full path of the `AuthServer` class and NOT `io.grpc.examples.authentication.AuthServer` as referenced in your auth-server script. Why this discrepancy?

This is blocking further review of this PR.